testenv/testenv.properties

```diff
@@ -10,13 +10,13 @@ ADOPTOPENJDK_SYSTEMTEST_REPO=https://github.com/adoptium/aqa-systemtest.git
 ADOPTOPENJDK_SYSTEMTEST_BRANCH=v1.0.0
 JDK8_REPO=https://github.com/adoptium/jdk8u.git
 JDK8_BRANCH_PREFIX=jdk8u402-b05
-JDK8_BRANCH=dev
+JDK8_BRANCH=jdk8u402-b05
 JDK11_REPO=https://github.com/adoptium/jdk11u.git
 JDK11_BRANCH_PREFIX=jdk-11.0.22+6
-JDK11_BRANCH=dev
+JDK11_BRANCH=jdk-11.0.22+6
 JDK17_REPO=https://github.com/adoptium/jdk17u.git
 JDK17_BRANCH_PREFIX=jdk-17.0.10+6
-JDK17_BRANCH=dev
+JDK17_BRANCH=jdk-17.0.10+6
 JDK21_REPO=https://github.com/adoptium/jdk21u.git
 JDK21_BRANCH_PREFIX=jdk-21.0.1
 JDK21_BRANCH=dev
```

Comment:
If you are intending to dry-run JDK21, you will also want to update JDK21_BRANCH to match what the BUILD pipeline will use.